.github/workflows/gradle-build.yml

```
@@ -0,0 +1,40 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Build & test with Gradle
+
+env:
+  SONATYPE_USER: ${{ secrets.SONATYPE_USER }}
+  SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+  PGP_SIGNING_KEY: ${{ secrets.PGP_SIGNING_KEY }}
+  PGP_SIGNING_PASSWORD: ${{ secrets.PGP_SIGNING_PASSWORD }}
+
+on:
+  push:
```

I think we want the following settings:

```suggestion
on:
  push:
    branches: [master]
  pull_request:
    branches: [master]
```

---

it is right config(suggested)!

---

Updated!